code/modules/clothing/suits/marine_armor.dm

```diff
@@ -697,3 +697,167 @@
+	name = "\improper Ballistic vest"
+	desc = "Civilian type armor, made to combat both melee and projectiles."
+	icon = 'icons/mob/clothing/suits/ert_suits.dmi'
+	item_icons = list(
+		slot_wear_suit_str = 'icons/mob/clothing/suits/ert_suits.dmi',
+	)
+	icon_state = "specops_vest"
+	soft_armor = list(MELEE = 30, BULLET = 50, LASER = 20, ENERGY = 25, BOMB = 30, BIO = 5, FIRE = 25, ACID = 30)
+	slowdown = SLOWDOWN_ARMOR_LIGHT
+	flags_armor_protection = CHEST|GROIN
+	flags_armor_features = NONE
+
+/obj/item/clothing/suit/storage/marine/specops/support
+	name = "\improper Ballistic vest"
+	desc = "Civilian type armor, made to combat both melee and projectiles."
+	icon_state = "specops_vest_drone"
+
+/obj/item/clothing/suit/storage/marine/specops/medic
+	name = "\improper Ballistic vest"
+	desc = "Civilian type armor, made to combat both melee and projectiles."
+	icon_state = "specops_vest_medic"
+
+/obj/item/clothing/suit/storage/marine/specops/leader
+	name = "\improper Ballistic vest"
+	desc = "Civilian type armor, made to combat both melee and projectiles."
+	icon_state = "specops_vest_leader"
+	slowdown = SLOWDOWN_ARMOR_MEDIUM
+
+//===========================MARINE VERSION================================
+
+/obj/item/clothing/suit/storage/marine/ballistic
+	name = "\improper Crasher multi-threat ballistic armor"
+	desc = "A reused design of a old body armor system from the 21st century."
+	soft_armor = list(MELEE = 30, BULLET = 50, LASER = 20, ENERGY = 25, BOMB = 30, BIO = 5, FIRE = 25, ACID = 30)
+	slowdown = SLOWDOWN_ARMOR_LIGHT
+	flags_armor_protection = CHEST|GROIN
+	icon = 'icons/mob/clothing/suits/marine_armor.dmi'
+	item_icons = list(
+		slot_wear_suit_str = 'icons/mob/clothing/suits/marine_armor.dmi'
+	)
+	icon_state = "ballistic_vest"
+	flags_armor_features = NONE
+
+	allowed = list(
+		/obj/item/weapon/gun,
+		/obj/item/tank/emergency_oxygen,
+		/obj/item/flashlight,
+		/obj/item/ammo_magazine,
+		/obj/item/storage/fancy/cigarettes,
+		/obj/item/tool/lighter,
+		/obj/item/weapon/baton,
+		/obj/item/restraints/handcuffs,
+		/obj/item/explosive/grenade,
+		/obj/item/binoculars,
+		/obj/item/weapon/combat_knife,
+		/obj/item/attachable/bayonetknife,
+		/obj/item/storage/belt/sparepouch,
+		/obj/item/storage/holster/blade,
+		/obj/item/storage/holster/belt,
+		/obj/item/weapon/energy/sword,
+	)
+
+/obj/item/clothing/suit/storage/marine/ballistic/jungle
+	name = "\improper Crasher multi-threat jungle ballistic armor"
+	desc = "A reused design of a old body armor system from the 21st century."
+	soft_armor = list(MELEE = 30, BULLET = 50, LASER = 20, ENERGY = 25, BOMB = 30, BIO = 5, FIRE = 25, ACID = 30)
+	slowdown = SLOWDOWN_ARMOR_LIGHT
+	flags_armor_protection = CHEST|GROIN
+	icon = 'icons/mob/clothing/suits/marine_armor.dmi'
+	item_icons = list(
+		slot_wear_suit_str = 'icons/mob/clothing/suits/marine_armor.dmi'
+	)
+	icon_state = "m_ballistic_vest"
+	flags_armor_features = NONE
+
+	allowed = list(
```

Comment:
Wouldn't this already be inherited from the base version of /obj/item/clothing/suit/storage/marine/ballistic ?